### PR TITLE
Release v1.10.0

### PR DESCRIPTION
**Pull request by adam-fowler**
CHANGELOG and package.json changes for v1.10.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+### Added
+
+- Support for swift-testing.
+- Toolchain selection command `swift: Select Toolchain...`.
+- Create new Swift project command `swift: Create New Project...`.
+- Use custom LSP requests for test discovery.
+- Support for recording multiple issues per test.
+- Code snippets for common actions, like creating tests, availability conditions, option sets.
+- CustomExecution for swift tasks.
+- UI showing Swift build status
```
Comment (award999) [inline CHANGELOG.md:14]: de-duplication of diagnostics

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+
+### Changed
+
+- Replace test coverage support with native VSCode coverage APIs.
+- Deprecate `swift.sourcekit-lsp.serverPath` setting.
```
Comment (award999) [inline CHANGELOG.md:19]: remove `swift.problemMatchCompileErrors` setting

File: package.json [hunk 3]
```diff
@@ -1149,7 +1149,7 @@
     "test": "find ./assets/test -type d -name '.build' -exec rm -rf {} + && find . -type d -name 'Package.resolved' -exec rm -rf {} + && tsc -p ./ && node ./out/test/runTest.js",
     "compile-tests": "find ./assets/test -type d -name '.build' -exec rm -rf {} + && npm run compile && npm run esbuild",
     "package": "vsce package",
-    "dev-package": "vsce package --no-update-package-json 1.9.1-dev",
+    "dev-package": "vsce package --no-update-package-json 1.10.1-dev",
```
Comment (award999) [inline package.json:1152]: 1.10.0?

Comment (adam-fowler) [reply package.json:1152]: That's for dev builds. It has to be later than the release build but earlier than what will be the next build.

File: CHANGELOG.md (hunk 1 above)
Comment (adam-fowler) [reply CHANGELOG.md:14]: It's in the fixed section.

File: package.json (hunk 3 above)
Comment (0xTim) [reply package.json:1152]: We'd still need to change it for minor version updates though. This is a pretty common pattern in extensions

**Review by 0xTim — APPROVED** [review]
LGTM

File: package.json (hunk 3 above)
Comment (adam-fowler) [reply package.json:1152]: It's a minor thing, I've updated it every release up until now. It'd be good to automate it but didn't find an easy way to do that

**Pull request merged**